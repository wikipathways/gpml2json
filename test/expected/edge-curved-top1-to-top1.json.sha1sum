56e748b37d4844cd7fb83536d46802162feb515b ?-
