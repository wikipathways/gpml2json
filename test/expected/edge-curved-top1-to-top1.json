{"pathway":{"@context":["https://cdn.rawgit.com/wikipathways/WpVocabularies/7a46a05/contexts/pvjs.jsonld"],"backgroundColor":"white","borderWidth":0,"color":"black","contains":["b26be","cf2f3","e3c5e"],"drawAs":"rect","gpmlElementName":"Pathway","height":156.90531138358756,"kaavioType":"Group","name":"test case","padding":5,"type":["Pathway"],"width":244.75876467964497,"x":0,"y":0,"zIndex":0,"fontSize":12,"fontWeight":"bold","textAlign":"left","verticalAlign":"top","dataSourceVersion":"20130813","standardName":"test case","displayName":"test case","textContent":"Name: test case","isSimilarTo":"http://wikipathways.org/index.php/Special:SearchPathways?query=test%20case&doSearch=1","id":"hash://sha256/a2520e7890a6ecff23acaa4bb7b2169a470621e23e1a68df907982973612471d"},"entityMap":{"cf2f3":{"id":"cf2f3","textContent":"GeneProduct","wpType":"GeneProduct","x":134.75876467964497,"y":93.76887234173131,"fontSize":10,"height":20,"verticalAlign":"middle","width":80,"zIndex":32966,"textAlign":"center","color":"#000000","backgroundColor":"#ffffff","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","borderWidth":1,"drawAs":"rect","lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["DataNode","SingleFreeNode","GeneProduct"],"kaavioType":"SingleFreeNode","gpmlElementName":"DataNode"},"e3c5e":{"id":"e3c5e","textContent":"0002","wpType":"GeneProduct","x":26.23680528517258,"y":106.90531138358756,"fontSize":10,"height":20,"verticalAlign":"middle","width":80,"zIndex":32967,"textAlign":"center","color":"#000000","backgroundColor":"#ffffff","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","borderWidth":1,"drawAs":"rect","lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["DataNode","SingleFreeNode","GeneProduct"],"kaavioType":"SingleFreeNode","gpmlElementName":"DataNode"},"b26be":{"id":"b26be","drawAs":"CurvedLine","borderWidth":1,"zIndex":32965,"points":[{"isAttachedTo":"e3c5e","attachmentDisplay":{"position":[0.5,0],"offset":[0,0]},"x":66.23680528517258,"y":106.90531138358756,"orientation":[0,-1]},{"x":120.49778498240877,"y":73.76887234173131},{"isAttachedTo":"cf2f3","attachmentDisplay":{"position":[0.5,0],"offset":[0,0]},"x":174.75876467964497,"y":93.76887234173131,"orientation":[0,1]}],"color":"#000000","type":["Interaction","Edge"],"kaavioType":"Edge","gpmlElementName":"Interaction","markerStart":"TBar","markerEnd":"Arrow","isAttachedTo":["e3c5e","cf2f3"]}}}
